fly/commands/ordering_pipeline.go

```diff
@@ -16,6 +17,7 @@ var ErrMissingPipelineName = errors.New("Need to specify atleast one pipeline na
 type OrderPipelinesCommand struct {
 	Alphabetical bool                       `short:"a"  long:"alphabetical" description:"Order all pipelines alphabetically"`
 	Pipelines    []flaghelpers.PipelineFlag `short:"p" long:"pipeline" description:"Name of pipeline to order"`
+	Team         string                     `long:"team" description:"Name of the team to which the pipeline, if different from the target default"`
 }
 
 func (command *OrderPipelinesCommand) Validate() (atc.OrderPipelinesRequest, error) {
```

Comment:
I noticed in the set-pipelines `Validate` (https://github.com/concourse/concourse/blob/master/fly/commands/set_pipeline.go#L43) there's a ValidateIdentifier check regarding the team.

@taylorsilva I defer this to you, since I'm not sure if it's needed in this case! (Sorry for tagging you again :) )

Reply:
I don't think this one may be needed. I expect a check like that to happen in the `set-team` command when creating/updating the team, but not here where it doesn't really have any affect on the team name itself. The team name isn't going to change from running `order-pipelines` :P